### DISCUSSION TITLE
[ccxt okx fetch_open_orders] this doesn't seem to work

### DISCUSSION THREAD

**Discussion by jupiter100** [community/community#53076] [Programming Help] [answered]
### Body

I want to fetch stop-loss orders and take-profit orders, but the fetch_open_orders function doesn't seem to be working.
(My personal information has been erased.)

```
symbol = 'SLP/USDT:USDT'
myokx.verbose = True

orders = myokx.fetch_open_orders(symbol)
```

I ran this function, but didn't receive any data.

![image](https://user-images.githubusercontent.com/107293276/232961464-3e2bfb06-19d0-4b36-afee-c9faf6ec8af9.png)

There are open orders like this, but ....

How can I get the fetch_open_orders function to work properly?
Please help me. Thank you

### Guidelines

- [X] I have read and understood this category's [guidelines](https://github.com/orgs/community/discussions/51384) before making this post.

**Comment by jupiter100** [comment, marked as answer]
I solved this problem
OrdType had to be set in params.

```
params = {'stop' : True,'ordType' : 'conditional'}
orders = myokx.fetch_open_orders(symbol, None, None, params)
```

works fine

**Reply by queenofcorgis** [reply]
Thanks for coming back with the solve ✅ and nice work on solving it yourself!
